tests/test_install.py

```diff
@@ -16,7 +16,13 @@
 '''
 
 yaml_without_version = f'''
-{PACKAGE}.core.filesystem.FileSystem:
+${PACKAGE}.core.filesystem.FileSystem:
+  host: ~/host
+  repository: repo
+'''
+
+yaml_with_package = f'''
+${PACKAGE}:{PACKAGE}.core.filesystem.FileSystem:
```

Comment:
The test would be better if package and module name would be different, otherwise the test might also succeed if the module name and not the package name is used by the code.

Reply:
Not to forget the most tricky part: the package name must not match the module name :)

Reply:
Above you used `pyyaml`.
Otherwise you could use https://github.com/hagenw/sphinxcontrib-katex, which has `sphinx` as it's only requirement.